README.md

```diff
@@ -197,94 +182,137 @@ coastal area.
+                           nsteps_minim = 5000,
+                           resources=machine)
+```
 
-Visualize the results: 
+But do not forget to kill your machines:
 ```python
-view = ouptut.render_interctive(representation= "ball+stick", add_backbone=True)
+
+machine.kill()
 ```
 
-This yields an interactive visualization of the protein's trajectory that can be visualized and manipulated in a standard jupyter notebook. The user can specify the representation used for the protein and choose to add the backbone to the visualization.  
-<p align="center">
-  <img width="900" height="300" src="https://github.com/inductiva/inductiva/assets/114397668/d8eb9c0b-8809-4da6-97a1-3448f2809af3">
-</p>
+## Async API
+
+Up until now, all examples have run synchronously, which allows users to get feedback about the simulation while it is running. However, this is not always the best option. For example, if the user wants to run a large number of simulations, it is better to run them asynchronously. This way, the user can launch all the simulations and then check the results when they are ready.
+
+Let's look at an example with the wind tunnel:
+
+```python
+from inductiva import fluids
 
+scenario = fluids.scenarios.WindTunnel(
+    flow_velocity=[30, 0, 0],
+    domain_geometry={"x": [-6, 12], "y": [-5, 5], "z": [0, 10]})
+
+# Path to a folder containing a set of objects
+objects_path = ["object_1.obj", "object_2.obj", ..., "object_1000.obj"]
+
+for object in objects_path:
+    task = scenario.simulate(object_path=object,
+                      simulation_time = 100,
+                      output_time_step = 50,
+                      resolution = "medium",
+                      async=True)
+```
+
+All of the simulations will be launched in one go. The user can check the status of the simulations and retrieve the results when they are ready. Check the FAQ section for more information on how to do this.
 
 ## Installation
 
 It is super simple to start using the API if you are already familiar with Python package management.
+
 One just needs to do
 ```
 pip install inductiva
 ```
 
 and your are good to go! You are ready to start [exploring our tutorial notebooks](https://github.com/inductiva/inductiva/tree/main/demos).
 
-Notice that, in a local computer you just need to do this once. When opening the tutorials in Google Colab, you may need to re-install this
-between notebooks.
+Notice that, in a local computer you just need to do this once. When opening the tutorials in Google Colab, you may need to re-install this between notebooks.
 
 ## API access tokens
 
-Please request your demo API token via the following Google Form (we will reply to you by email):
+To use **Inductiva API** you will need an API token to log in. Please request your demo API token via the following Google Form (we will reply to you by email):
```

Comment:
Remove the bit "to log in". You don't actually log in, so this may be misleading.